app/theme/client/imports/forms/input.css

```diff
@@ -5,6 +5,23 @@ textarea.rc-input__element {
 	font-family: inherit;
 }
 
+.rc-input__password {
+	display: flex;
+	align-items: center;
+	border-width: var(--input-border-width);
+	border-color: var(--input-border-color);
+
+	& > .rc-input__element {
+		width: 90%;
+		border: 0;
+	}
+
+	& > .pass-icon {
+		padding: .5em;
+		width: 10%;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "width" to come before "padding" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4969942)